packages/regl-worldview/src/commands/GLText.js

```diff
@@ -241,6 +249,7 @@ function makeTextCommand() {
         billboard: (ctx, props) => ({ buffer: props.billboard, divisor: 1 }),
         foregroundColor: (ctx, props) => ({ buffer: props.foregroundColor, divisor: 1 }),
         backgroundColor: (ctx, props) => ({ buffer: props.backgroundColor, divisor: 1 }),
+        highlightColor: (ctx, props) => ({ buffer: props.highlightColor, divisor: 1 }),
```

Comment:
With this change, maybe we could remove `enableHighlight` React prop since we can tell by the presence of `highlightColor` prop?  In addition, maybe add an `autoHighlight` prop which computes the contrasted color based on the text marker's color so that the consumers don't have to manually compute highlight colors?

Reply:
`enableHighlight` is a webgl prop (not react) -- the only prop react is passing is the indices in the string it needs to highlight. I may be able to co-op the existence of the highlight color in the frag shader, not sure if GLSL has a good mechanism for truthiness, slash not sure if this will make things more readable. 

RE: auto-computing highlight colors -- that's an interesting idea! I'll take a look and see if it's an easy enough lift for this PR

Reply:
> RE: auto-computing highlight colors -- that's an interesting idea! I'll take a look and see if it's an easy enough lift for this PR

I forgot to mention -- by default I set the font color (`foregroundColor`) to black when it's highlighted, so in a sense it is autocomputed already for consumers. This is how it is done for web pages in Chrome when you `control - f`, so I think I'd like to keep it the way it is. Lemme know what you think!